lib/install.js

```diff
@@ -397,13 +409,15 @@ function install(opts, cb) {
     var yauzl = require('yauzl');
     var extractPath = path.join(path.dirname(zipFilePath), path.basename(zipFilePath, '.zip'));
 
-    yauzl.open(zipFilePath, {lazyEntries: true}, function onOpenZipFile(err, zipFile) {
+    yauzl.open(zipFilePath, function onOpenZipFile(err, zipFile) {
       if (err) {
         cb(err);
         return;
       }
-      zipFile.readEntry();
-      zipFile.once('entry', function (entry) {
+      zipFile.on('entry', function (entry) {
+        if (/.*\/.*/.test(entry.fileName)) {
```

Comment:
Could this have a negative impact on other drivers?

Reply:
Not that I'm aware of, I looked into the `yauzl` documentation, their guides state that this is a valid check for folders inside zip files. The only negative side-effect would be if some browsers suddenly start putting their webdrivers inside folders in the zip, but if that happens we'd need to make some changes in the code anyway.

Reply:
👍